api/connectors/cortex/job.md

```diff
@@ -0,0 +1,35 @@
+# Job
+
+## Model definition
+
+Required attributes:
+- `analyzerId` (string): identifier of the analyzer used by the job
+- `status` (enumeration): status of the job (`InProgress`, `Success`, `Failure`) **default=`InProgress`**
```

Comment:
This is an internal field managed by the backend. It's be better to list it under a `Attributes generated by the backend` section.